### ISSUE TITLE
Heatbed/Fan reversed with hotend in RAMPS 1.4 EFF config 1.10-RC7

### ISSUE THREAD

**Issue by moo1426** [MarlinFirmware/Marlin#4462]
When I enable my hotend with my board defined as a RAMPS 1.4 EFF the firmware enables the fan connected to the heatbed/fan2 terminal. The error is fixed if the board is defined as RAMPS 1.4 EFB. I am using singlenozzle if it makes a difference on 1.10-RC7.

**Comment by thinkyhead** [comment]
Hmm, that's odd. Both EFF and EFB define `FAN_PIN` as pin 9, which I believe is the middle connector.
```cpp
#if MB(RAMPS_14_EFF) || MB(RAMPS_13_EFF) || ENABLED(IS_RAMPS_EFB)
  #define FAN_PIN           9 // (Sprinter config)
  . . .
#endif
```

**Comment by thinkyhead** [comment]
You must be referring to `FAN1_PIN` and not `FAN_PIN`. I ran into this just now as I was putting together #4463. Since this is a pre-release and in previous versions of Marlin you had to self-assign the `FAN1_PIN`, I think this is probably safe to patch. I had been concerned that setups wired backward would be a problem, but this is new to 1.1.0 so I imagine it must be quite rare.

**Comment by boelle** [comment]
if its safe to patch why not fix and close this one to get the list shorter?

**Comment by thinkyhead** [comment]
@boelle See the above linked merged PR #4463. Still hoping for feedback from @moo1426.
